tools/openshift-ci/Dockerfile.centos

```diff
@@ -25,7 +23,4 @@ RUN set -x && \
 RUN scl enable rh-ror50 /bushslicer/tools/install_os_deps.sh
```

Comment:
@droslean are you checking it out as `bushslicer` though?

Reply:
@droslean that is, are you checking out the bushslicer repo into a directory called bushslicer? or to a directory called verification-tests? probably not much of a change to update the dir here and on line 26 to verification-tests for consistency.

Reply:
@droslean, just wondering if you git clone it to a name as `bushslicer` or just follow the repo name.  The command would fail run if you just follow the repo name since the name has been changed to `verification-tests` in github.

Reply:
I am not cloning anything here. I just take the parent directory and add it temporarily in the image. then delete it.

Reply:
@droslean, how does the parent directory get created?